all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md

```diff
@@ -0,0 +1,18 @@
+# Cell-health CellProfiler features with `cp_process`
+
+## Introduction
+
+Here, we benchmark the `cp_process_singlecells` workflow with the NF1 dataset [dataset](https://github.com/WayScience/nf1_cellpainting_data) that is demonstrated in the jupyter notebook.
+
+## About the data
+
+The NF1 data repository, available at [https://github.com/WayScience/nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data), houses image-based profiles assayed through CellPainting. This dataset comprises information from four plates, each annotated with metadata corresponding to specific genotypes. Genotypes are categorized as wild type (+/+) and Null (-/-).
```

Comment:
Might be worth expanding on what format the image-based profile is. Both in terms of file type and possibly structure. Might be out of scope.

Reply:
Good point, but this might be out of the scope.